### PR TITLE
Issue 59 - Added platform_name and synchronized definition order

### PR DESCRIPTION
**Pull request by jo-asplin-met-no**

### CODE REVIEW

**Comment by Jeffrey-Vervoort-KNMI** [comment]
Looks good to me.

**Review by Jeffrey-Vervoort-KNMI — APPROVED** [review]

File: datastore/migrate/data/migrations/1709282607_initialise_schema.up.sql [hunk 1]
```diff
@@ -14,19 +14,7 @@ CREATE TABLE time_series (
 	-- --- END non-string metadata -----------------
 
 	-- --- BEGIN string metadata (handleable with reflection) -----------------
-
```
Comment (lukas-phaf) [inline datastore/migrate/data/migrations/1709282607_initialise_schema.up.sql:17]: PR looks good, but please discuss with Amund before merging...

**Review by lukas-phaf — COMMENTED** [review]
This looks fine, but please discuss with Amund before merging...

File: datastore/migrate/data/migrations/1709282607_initialise_schema.up.sql (hunk 1 above)
Comment (jo-asplin-met-no) [reply datastore/migrate/data/migrations/1709282607_initialise_schema.up.sql:17]: Agreed.

**Review by lukas-phaf — APPROVED** [review]

**Pull request merged**